packages/design-system/src/components/Table/TableHeader.jsx

```diff
@@ -0,0 +1,97 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+
+export const TableHeader = (props) => {
+  const {
+    title,
+    type,
+    width,
+    scope,
+    className,
+    stackedTitle,
+    stackedClassName,
+    ...attributeOptions
+  } = props;
+
+  const classes = classNames(
+    'ds-c-table__header',
+    className,
+    type === 'numeric' ? 'ds-c-table__header--numeric' : null,
+    width ? 'ds-c-table__header--width-' + width : null
+  );
+
+  const stackedClasses = classNames(
+    'ds-c-table--stacked__col-header',
+    'ds-u-font-weight--bold',
+    stackedClassName
+  );
+
+  const renderStackedTitle = () => {
+    const isValidStackedTitle = stackedTitle && stackedTitle.length > 0;
+
+    return (
+      { isValidStackedTitle } && (
+        <span aria-hidden="true" className={stackedClasses}>
+          {stackedTitle}
+        </span>
+      )
+    );
+  };
+
+  return (
+    <th
+      className={classes}
+      role={scope === 'col' ? 'columnheader' : 'rowheader'}
+      scope={scope}
+      {...attributeOptions}
+    >
+      {renderStackedTitle()}
+      {title}
+    </th>
+  );
+};
+
+TableHeader.defaultProps = {
+  className: '',
```

Comment:
I don't think any of these default `className` or `stackedClassName` props are necessary.